platform/broadcom/sonic-platform-modules-delta/debian/files

```diff
@@ -1,2 +1,3 @@
 platform-modules-ag9032v1_1.1_amd64.deb main extra
 platform-modules-ag9064_1.1_amd64.deb main extra
+platform-modules-ag5648_1.1_amd64.deb main extra
```

Comment:
this is auto-generated file, please do not check-in this file.

Reply:
can you remove this file?

Reply:
In #1481 . I removed it &  all *.cmd files

Reply:
why create a new PR? why not update this PR, it is easier for me to check if all comments are addressed. When you move to a new PR, there is no history, no context, for me it is a completely new PR. I have to start from scratch to review.

Reply:
OK! I will update on this PR.